airgun/views/architecture.py

```diff
@@ -6,7 +6,9 @@
 class ArchitectureView(View):
     title = Text("//h1[text()='Architectures']")
     new = Text("//a[contains(@href, '/architectures/new')]")
-    navigate_locator = "//a[@id='menu_item_architectures']"
+    page_navigate_locator = "//a[@id='menu_item_architectures']"
+    entity_navigate_locator = (
+        "//a[contains(@href, 'edit') and contains(@href, 'arch')]")
```

Comment:
You can use Text widget to represent links. WT is designed to avoid arbitrary locators outside of widgets (except when needed to be passed as a parameter to the widget).

Reply:
Why does it have such name? It is so bad designed to use widget Text for Links and for Buttons for example.
Okay, I will check that out

Reply:
If you think about it, a plain link is text. Of course, buttons are specific, but still :)